recipe/run_conda_forge_build_setup_win.bat

```diff
@@ -91,15 +91,23 @@ for %%i in ("%~dp0.") do set "SCRIPT_DIR=%%~fi"
 <cuda.version set /p CUDA_VERSION=
 del cuda.version
 if not "%CUDA_VERSION%" == "None" (
-    call "%SCRIPT_DIR%\install_cuda.bat" %CUDA_VERSION%
-    if errorlevel 1 (
-        echo Could not install CUDA
-        exit 1
+    if "%CUDA_VERSION:~0,2%" == "12" (
+        :: Don't call install_cuda, as we'll get CUDA packages from CF
+        set "CUDA_PATH="
+        set "CONDA_OVERRIDE_CUDA=%CUDA_VERSION%"
+        :: Export CONDA_OVERRIDE_CUDA to allow __cuda to be detected on CI systems without GPUs
+        echo set "CONDA_OVERRIDE_CUDA=%CONDA_OVERRIDE_CUDA%" >> "%CONDA_PREFIX%\etc\conda\activate.d\conda-forge-ci-setup-activate.bat"
```

Comment:
Why do we need this, too, in the 12.x case? It also looks like it doesn't really do anything as it sets the variable to itself.

Reply:
I thought the comment is self-explanatory? Also, this treatment is copied from existing code (see line 133-134 below), I didn't reinvent any wheel 🙂

Reply:
> I thought the comment is self-explanatory?

Not fully. Reading the source again, it makes sense. "Bake-in" instead of "Export" would have made it clear to me but probably not to everyone else. Happy to keep it as-is.

Reply:
Basically with how CUDA 11 is handled (both on Linux and Windows), most CUDA packages come from the system. There is a bunch of logic here to handle those CUDA 11 cases

With CUDA 12, we now have Conda packages and so don't need to use that logic. Eventually we could drop the CUDA 11 logic from here when we drop CUDA 11. In the interim, we just need to make sure we "do nothing" on CUDA 12